elements/rh-progress-stepper/rh-progress-stepper.css

```diff
@@ -0,0 +1,144 @@
+:host {
+  --stepLineColor: var(--rh-color-border-subtle);
+
+  display: grid;
+  row-gap: var(--rh-length-lg, 16px);
+
+  & slot {
+    padding: 0;
+    display: grid;
+    place-content: baseline normal;
+    grid-auto-columns: 1fr;
+  }
+
+  & #current-step {
+    /* Visually Hiding */
+    block-size: 1px;
+    border: 0;
+    clip: rect(0, 0, 0, 0);
+    inline-size: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    white-space: nowrap;
+
+    /* End Visual Hiding */
+    font-weight: var(--rh-font-weight-body-text-medium, 500);
+    text-decoration:
+      underline
+      dotted
+      var(--rh-color-text-secondary,
```

Comment:
remind me @zeroedin what was our conclusion, that these need fallbacks in element css? we can rely on base-theme being there, so I think we can remove these